dev-haskell/fourmolu/fourmolu-0.3.0.0.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="A formatter for Haskell source code"
+HOMEPAGE="https://github.com/parsonsmatt/fourmolu"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+IUSE="examples executable"
+
+PATCHES=( "${FILESDIR}/${P}-add-flags.patch" )
+
+RDEPEND=">=dev-haskell/aeson-1.5.2:=[profile?] <dev-haskell/aeson-1.6:=[profile?]
+	>=dev-haskell/dlist-0.8:=[profile?] <dev-haskell/dlist-2.0:=[profile?]
+	>=dev-haskell/exceptions-0.6:=[profile?] <dev-haskell/exceptions-0.11:=[profile?]
+	>=dev-haskell/ghc-lib-parser-8.10:=[profile?] <dev-haskell/ghc-lib-parser-8.11:=[profile?]
+	>=dev-haskell/hsyaml-0.2:=[profile?] <dev-haskell/hsyaml-0.3:=[profile?]
+	>=dev-haskell/hsyaml-aeson-0.2:=[profile?] <dev-haskell/hsyaml-aeson-0.3:=[profile?]
+	>=dev-haskell/mtl-2.0:=[profile?] <dev-haskell/mtl-3.0:=[profile?]
+	>=dev-haskell/syb-0.7:=[profile?] <dev-haskell/syb-0.8:=[profile?]
+	>=dev-haskell/text-0.2:=[profile?] <dev-haskell/text-1.3:=[profile?]
+	>=dev-lang/ghc-8.6.3:=
+	executable? ( >=dev-haskell/gitrev-1.3:=[profile?] <dev-haskell/gitrev-1.4:=[profile?]
+			>=dev-haskell/optparse-applicative-0.14:=[profile?] <dev-haskell/optparse-applicative-0.17:=[profile?] )
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-2.4.0.1
+	test? ( >=dev-haskell/hspec-2.0 <dev-haskell/hspec-3.0
+		>=dev-haskell/path-0.6
+		>=dev-haskell/path-io-1.4.2 <dev-haskell/path-io-2.0 )
+"
+
+src_prepare() {
+	default
+	cabal_chdeps\
+		'path >=0.6 && <0.8' 'path >=0.6'
+}
+
+src_configure() {
+	haskell-cabal_src_configure \
+		--flag=-dev \
+		$(cabal_flag examples examples)\
+		$(cabal_flag executable executable)
+}
+
+# Hacky, but gets the job done. Will improve on this if I find out how...
```

Comment:
I opted for a conditional patch on the `.cabal` file, since there are so many example files included (~1100). I also asked upstream to make these example files optional, possibly by creating new `fourmolu-examples`/`ormolu-examples` packages:

- parsonsmatt/fourmolu#59
- tweag/ormolu#682